packages/hardhat-core/sample-projects/advanced-ts/hardhat.config.ts

```diff
@@ -0,0 +1,45 @@
+import * as dotenv from "dotenv";
+
+import { HardhatUserConfig, task } from "hardhat/config";
+import "@nomiclabs/hardhat-etherscan";
+import "@nomiclabs/hardhat-waffle";
+import "@typechain/hardhat";
+import "hardhat-gas-reporter";
+import "solidity-coverage";
+
+dotenv.config();
+
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (const account of accounts) {
+    console.log(account.address);
+  }
+});
+
+// You need to export an object to set up your config
+// Go to https://hardhat.org/config/ to learn more
+
+const config: HardhatUserConfig = {
+  solidity: "0.8.4",
+  networks: {
+    hardhat: {
+      initialBaseFeePerGas: 0, // workaround from https://github.com/sc-forks/solidity-coverage/issues/652#issuecomment-896330136 . Remove when that issue is closed.
+    },
+    ropsten: {
+      url: process.env.ROPSTEN_URL || "",
+      accounts: [process.env.PRIVATE_KEY || `0x${"0".repeat(40)}`],
```

Comment:
```suggestion
      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
```

Reply:
also check the js version

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1770/commits/91d120316dc2d9eded7848b61258105139ab7493